src/OrbitGl/PresetFile.cpp

```diff
@@ -4,12 +4,46 @@
 
 #include "PresetFile.h"
 
+#include <absl/strings/match.h>
+#include <google/protobuf/io/zero_copy_stream_impl_lite.h>
+#include <google/protobuf/text_format.h>
+
+#include "OrbitBase/File.h"
 #include "OrbitBase/Logging.h"
+#include "OrbitBase/ReadFileToString.h"
 
 namespace orbit_gl {
```

Comment:
Not related to this change, but what's the reason for this to be in `OrbitGl`?

Reply:
I guess it wants to be its own component... I will create a followup CL for this